<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1640544609503' stop='1640544610616'>
    <name>Test New Scenario </name>
    <title>Test New Scenario </title>
    <test-cases>
        <test-case start='1640544609504' status='failed' stop='1640544610615'>
            <name>Test Hello World</name>
            <title>Test Hello World</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
            <failure>
                <message>HTTP status 200 !== 404</message>
                <stack-trace>AssertionError [ERR_ASSERTION]: HTTP status 200 !== 404
                    at Expect._validateStatus (node_modules/pactum/src/models/expect.js:100:14)
                    at Expect.validate (node_modules/pactum/src/models/expect.js:45:10)
                    at Tosser.validateResponse (node_modules/pactum/src/models/Tosser.js:225:23)
                    at Tosser.validate (node_modules/pactum/src/models/Tosser.js:192:18)
                    at Tosser.toss (node_modules/pactum/src/models/Tosser.js:42:18)
                    at processTicksAndRejections (node:internal/process/task_queues:96:5)</stack-trace>
            </failure>
            <description type='text'>Now in TEST</description>
        </test-case>
    </test-cases>
</ns2:test-suite>